src/test/read-parameter-defaults.ts

```diff
@@ -0,0 +1,9 @@
+export function readParameterDefaults<T>(schema: any): T {
```

Comment:
Maybe we could create an Interface for json schemas? They always follow this pattern, don't they:
```typescript
interface {
  $schema: string;
  properties: object;
}
```

Reply:
I introduced an interface for schematic.json files. 👍